lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -116,9 +116,33 @@ def void(authorization, options = {})
       end
 
       def verify(credit_card, options = {})
-        MultiResponse.run(:use_first_response) do |r|
-          r.process { authorize(100, credit_card, options) }
-          r.process(:ignore_result) { void(r.authorization, options) }
+        if options[:allow_card_verification] == true
+          options.delete(:allow_card_verification)
+          exp_month = credit_card.month.to_s
+          exp_year = credit_card.year.to_s
+          expiration = "#{exp_month}/#{exp_year}"
+          payload = {
+            creditCard: {
+              number: credit_card.number,
+              expirationDate: expiration,
```

Comment:
should this be `expiration_date`? That's what I'm seeing in the Braintree docs...

Reply:
Yes! Good catch - revised!

Reply:
I also revised `postalCode` to be `postal_code` 👍